lib/testium-driver-wd.js

```diff
@@ -164,9 +164,17 @@ function initDriver(testium) {
       throw new Error(`invalid expectedStatusCode option: ${code}`);
     }
 
+    const waitForLoadEvent = options.waitForLoadEvent !== false;
+
+    let waitForLoadEventOrNull = function() {};
+    if (waitForLoadEvent) {
+      waitForLoadEventOrNull = this.waitForLoadEvent;
+    }
+
     return this.navigateTo(url, options)
       .getStatusCode()
-      .then(assertCode);
+      .then(assertCode)
+      .then(() => waitForLoadEventOrNull.call(this));
```

Comment:
Would it be easier to make this `() => waitForLoadEvent && this.waitForLoadEvent()`? Creating two temporary functions seems a bit excessive. :)

Reply:
sure - was originally trying to follow a similar pattern to the `assertStatusCode` section, but I see that they are not quite the same